.vscode/launch.json

```diff
@@ -6,7 +6,29 @@
             "type": "node",
             "request": "attach",
             "port": 9229,
-            "preLaunchTask": "func: host start"
+            /*
+                Previous to run this, we need to run the functions manually
+                using azure-functions-core-tools (npm install -g azure-functions-core-tools).
+
+                Go to packages/services or packages/worker and run:
+
+                func start --port 9229
+            */
+            "processId": "${command:PickProcess}"
```

Comment:
How does manually starting the function affect debugging? Do we still hit breakpoints in VS Code?

Reply:
Yes, we still hit breakpoints in VS Code. The only difference is that you need to run the function manually first.

I didn't find another way to lunch the function from a subfolder.

Reply:
Ah, ok. Yes, it's very picky with folders :/